webrtc-stats.html

```diff
@@ -924,7 +926,8 @@
               </dt>
               <dd>
                 <p>
-                  Represents the number of unique datachannels opened.
+                  Represents the number of unique DataChannels returned from a successful
+                  createDataChannel() call.
```

Comment:
The data channel returned from createDataChannel may not be initially opened, though. Do we need separate "dataChannelsCreated"/"dataChannelsOpened" count?